src/components/Calculator.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿